helix-term/src/commands.rs

```diff
@@ -3466,7 +3466,7 @@ pub enum Open {
     Above,
 }
 
-fn open(cx: &mut Context, open: Open) {
+fn open(cx: &mut Context, open: Open, add_comment_tokens: bool) {
```

Comment:
For these parameters we usually prefer enums for their readability, see `YankAction` or `Operation`